developer_manual/client_apis/OCS/ocs-api-overview.rst

```diff
@@ -166,13 +166,19 @@ Values of the theming app are exposed through the capabilities API, allowing cli
 		<slogan>A safe home for all your data</slogan>
 		<color>#0082c9</color>
 		<color-text>#ffffff</color-text>
-		<color-element>#006295</color-element>
+		<color-element>#0082c9</color-element>
+		<color-element-bright>#aaaaaa</color-element-bright>
+		<color-element-dark>#555555</color-element-dark>
 		<logo>http://cloud.example.com/index.php/apps/theming/logo?v=1</logo>
 		<background>http://cloud.example.com/index.php/apps/theming/logo?v=1</background>
 		<background-plain></background-plain>
 		<background-default></background-default>
 	</theming>
 
+For elements like radio buttons, input borders and more the, instead of the primary ``color`` value, ``color-element-bright` should be used on bright background, and ``color-element-dark`` on dark background.
```

Comment:
```suggestion
For elements like radio buttons, input borders and more, instead of the primary ``color`` value, the ``color-element-bright`` should be used on bright background, and ``color-element-dark`` on dark background.
```

Reply:
fixed